features/filter_by_pubkey.feature

```diff
@@ -0,0 +1,149 @@
+      | per_page     | 50    |
+
+  Scenario: Filter updates in real-time when new items are published
+    Given I am viewing filtered results for "npub1alice789xyz..."
+    And the results show 3 items
+    When "npub1alice789xyz..." publishes a new TASK "Backend Integration"
+    Then the filtered results should automatically update
+    And I should now see 4 items in the results
+    And the new task should appear in the list
+
+  Scenario: Clear pubkey filter to view all items
+    Given I have filtered items by pubkey "npub1alice789xyz..."
+    And I see 3 filtered results
+    When I clear the pubkey filter
+    Then I should see all 7 items
+    And items from all publishers should be visible
+
+  Scenario Outline: Filter by pubkey across different item types
+    When I filter items by pubkey "<pubkey>"
+    And I filter by item type "<item_type>"
+    Then I should see <count> items in the results
+
+    Examples:
+      | pubkey                | item_type | count |
+      | npub1alice789xyz...   | PROJECT   | 1     |
+      | npub1alice789xyz...   | TASK      | 1     |
+      | npub1alice789xyz...   | BOUNTY    | 1     |
+      | npub1bob456def...     | PROJECT   | 1     |
+      | npub1bob456def...     | TASK      | 1     |
+      | npub1bob456def...     | BOUNTY    | 0     |
+      | npub1charlie123abc... | PROJECT   | 1     |
+      | npub1charlie123abc... | TASK      | 0     |
+      | npub1charlie123abc... | BOUNTY    | 1     |
+
+  # UI/UX Scenarios
+
+  Scenario: Display publisher information in filter UI
+    When I open the filter options
+    Then I should see a "Filter by Publisher" section
+    And I should see a searchable dropdown with registered publishers:
+      | display                        | npub                  |
+      | PillarOne (npub1alice789xyz...)   | npub1alice789xyz...   |
+      | PillarTwo (npub1bob456def...)     | npub1bob456def...     |
+      | PillarThree (npub1charlie123abc...)| npub1charlie123abc... |
+
+  Scenario: Autocomplete publisher search
+    Given I am in the publisher filter input field
+    When I type "Pillar"
+    Then I should see autocomplete suggestions:
+      | suggestion                     |
+      | PillarOne (npub1alice789xyz...)   |
+    When I select "PillarOne (npub1alice789xyz...)"
+    Then the filter should be applied with "npub1alice789xyz..."
+
+  Scenario: Persist filter selections across sessions
+    Given I have filtered items by pubkey "npub1alice789xyz..."
+    When I refresh the page
+    Then the pubkey filter "npub1alice789xyz..." should still be active
+    And I should still see the filtered results
+
+  Scenario: Show active filters clearly in the UI
+    When I filter items by pubkey "npub1alice789xyz..."
+    Then I should see an active filter badge showing "Publisher: PillarOne"
+    And the badge should have a clear option to remove the filter
+
+  # Performance Scenarios
+
+  Scenario: Filter performance with large dataset
```

Comment:
Hmm performance specs here are tricky. Not sure how much it makes sense to have them.

If we were a building a SaaS, then we would probably have a prod-like staging environment where a test like "load within 2 seconds under some load" could be defined and measured.
But the idea with the incubator is that multiple people can run them and index off of whatever relays they choose.
So I don't think this is test that we would ever automate.
And the idea that "pages should load in a reasonable time" is probably a general enough implied requirement that we don't need to specify it everywhere.
I would probably remove this one.